scripts/keyboard_node.py

```diff
@@ -176,12 +185,13 @@ def main(debug=DEFAULT_DEBUG):
         pygame.display.flip()
 
         if using_joystick:
-            txt = "X: {} Y: {}".format(int(axis[0]*SCALE), int(axis[1]*SCALE))
+            txt = "Linear: {} Angular: {}".format(int(axis[1]*SCALE), int(axis[0]*SCALE))
```

Comment:
Dá pra usar f-strings

```suggestion
            txt = f"Linear: {int(axis[1]*SCALE)} Angular: {int(axis[0]*SCALE)}"
```